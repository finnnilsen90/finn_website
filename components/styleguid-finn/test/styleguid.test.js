const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer');
const jest = require('jest');
const sinon = require('sinon');
let { shallow, mount, render } = require('enzyme');

const Styleguid = require('../Styleguid-finn');

let StyleguidConst = new Styleguid(); 

describe('component connection', () => {

    it('should be defined', () => {
        expect(Styleguid).toBeDefined();
    })

})  
   
describe('<Styleguid />', () => {

    const wrapper = shallow(<Styleguid />);

    it('renders components', () => {
        expect(wrapper).toMatchSnapshot();
    });

    // const component = mount(<Styleguid />);

    // it('should turn to green on click', () => {
    //     // component.find('.hamburger').simulate('click');
    //     // expect(component).toMatchSnapshot();
    //     sinon.spy(Styleguid.prototype, 'componentDidMount');
    //     const component = mount(<Styleguid />);
    //     expect(Styleguid.prototype.componentDidMount).to.have.property('callCount', 1);
    // });  

})
