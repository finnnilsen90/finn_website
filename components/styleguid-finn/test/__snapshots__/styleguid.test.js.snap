// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Styleguid /> renders components 1`] = `
<div
  className="containter font"
>
  <Color />
  <Libcomp />
  <div
    className="footer"
  >
    <Footer />
  </div>
</div>
`;
